packages/fast-development-site-react/src/components/toc/index.tsx

```diff
@@ -17,6 +17,7 @@ const style: ComponentStyles<ITocManagedClasses, IDevSiteDesignSystem> = {
     toc: {
         margin: "0",
         padding: "0",
+        fontSize: "14px",
```

Comment:
this is more of a question for @Falkicon - This isn't on our type ramp; is this a planned update to the type ramp or is this difference intentional?